### PR TITLE
Login register

### PR DESCRIPTION
**Pull request by kobrynsky**

### CODE REVIEW

File: src/containers/LoginFormContainer.js [hunk 1]
```diff
@@ -27,9 +29,12 @@ export default class LoginFormContainer extends Component {
       lastName: response.last_name,
       email: response.email,
       picture: response.picture.data.url,
-      token: response.accessToken
+      token: response.accessToken,
+      externalProviderName: 'Facebook',
```
Comment (jskurzynska) [inline src/containers/LoginFormContainer.js]: I think it should be const value, reusable

File: src/containers/LoginFormContainer.js [hunk 2]
```diff
@@ -38,47 +43,50 @@ export default class LoginFormContainer extends Component {
...
-      userID: response.profileObj.googleId,
+      token: response.accessToken,
       firstName: response.profileObj.givenName,
       lastName: response.profileObj.familyName,
       email: response.profileObj.email,
       birthday: response.birthday,
-      picture: response.profileObj.imageUrl
+      picture: response.profileObj.imageUrl,
+      externalProviderName: 'Google',
+      roleName: 'Normal'
```
Comment (jskurzynska) [inline src/containers/LoginFormContainer.js]: Normal?

File: src/containers/LoginFormContainer.js [hunk 3]
```diff
@@ -38,47 +43,50 @@ export default class LoginFormContainer extends Component {
...
       isLoggedIn: true,
-      userID: response.profileObj.googleId,
+      token: response.accessToken,
       firstName: response.profileObj.givenName,
       lastName: response.profileObj.familyName,
       email: response.profileObj.email,
       birthday: response.birthday,
-      picture: response.profileObj.imageUrl
+      picture: response.profileObj.imageUrl,
+      externalProviderName: 'Google',
```
Comment (jskurzynska) [inline src/containers/LoginFormContainer.js]: it also should be const value

File: src/containers/LoginFormContainer.js [hunk 4]
```diff
@@ -38,47 +43,50 @@ export default class LoginFormContainer extends Component {
...
+    this.addNewUser();
   };
 
   addNewUser = () => {
-    axios
-      .post(`${BASE_URL}/login`, {
-        firstName: this.state.first_name,
-        lastName: this.state.last_name,
+    axios.post(`${BASE_URL}/login`, {
+        firstName: this.state.firstName,
```
Comment (jskurzynska) [inline src/containers/LoginFormContainer.js]: you can use destructuring state, code will look better:
**const { picture, firstName} = this.state;** 
and below you can use just picture without repeating this.state
please do it with all properties used here

File: src/containers/LoginFormContainer.js [hunk 5]
```diff
@@ -38,47 +43,50 @@ export default class LoginFormContainer extends Component {
...
+        photoUrl: this.state.picture,
+        externalProviderToken: this.state.token,
+        externalProviderName: this.state.externalProviderName,
+        roleName: this.state.roleName
+
       })
-      .then(() => {
-        this.props.history.push("/");
+      .then((response) => {
+        console.log(response);
```
Comment (jskurzynska) [inline src/containers/LoginFormContainer.js]: do we need it or it was jus for your tests?

File: src/containers/LoginFormContainer.js [hunk 6]
```diff
@@ -38,47 +43,50 @@ export default class LoginFormContainer extends Component {
...
+        if(response.status === 200)
+          this.props.history.push("/");
+        else if(response.status === 201)
+          this.props.history.push("/editprofile");
```
Comment (jskurzynska) [inline src/containers/LoginFormContainer.js]: I would suggest const value (it's used in router as well)

File: src/containers/LoginFormContainer.js [hunk 7]
```diff
@@ -38,47 +43,50 @@ export default class LoginFormContainer extends Component {
...
+        else
+          console.log("Something went wrong...");
       })
       .catch(err => console.log(err));
   };
 
   render() {
-    const { isLoggedIn, picture, firstName, lastName, email } = this.state;
+    const { isLoggedIn } = this.state;
```
Comment (jskurzynska) [inline src/containers/LoginFormContainer.js:86]: ❤️

**Review by Bartosz97 — APPROVED** [review]

**Pull request merged**